recipes/flash-attn/conda-forge.yml

```diff
@@ -0,0 +1,2 @@
+azure:
+  timeout_minutes: 360
```

Comment:
Hmm, it seems that the azure builds still timeout after 30-40min (e.g. at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=927858&view=logs&jobId=67448ffb-e003-5bfa-c062-cee3af60fcba&j=67448ffb-e003-5bfa-c062-cee3af60fcba&t=818ff20d-11b7-59db-6ce1-bb4df921454a). Maybe this only works on the feedstock repo?

Reply:
Keep this file because we need to have it in the feedstock.